process/http.go

```diff
@@ -147,9 +147,9 @@ func (p procHTTP) Apply(ctx context.Context, capsule config.Capsule) (config.Cap
 	url := p.Options.URL
 	if strings.Contains(url, httpInterp) {
 		if p.Key != "" {
-			url = strings.Replace(url, httpInterp, capsule.Get(p.Key).String(), 1)
+			url = strings.Replace(url, httpInterp, capsule.Get(p.Key).String(), -1)
```

Comment:
style: `strings.ReplaceAll` ([docs](https://pkg.go.dev/strings#ReplaceAll)) should be semantically equivalent as this code but has a slightly more clear API name